dev/test-studio/plugins/locale-no-nb/bundles/studio.ts

```diff
@@ -602,6 +602,116 @@ const studioResources: Record<StudioLocaleResourceKeys, string> = {
+  'asset-source.dialog.accept-message':
+    'Viser kun media av godkjente typer: <strong>{acceptTypes}</strong>',
+  /** Text shown when selecting an image but there's no images to select from */
+  'asset-source.dialog.no-assets_image': 'Ingen bilder',
+  /** Text shown when selecting a file but there's no files to select from */
+  'asset-source.dialog.no-assets_file': 'Ingen filer',
+  /** Menu item for showing where a particular asset is used */
+  'asset-source.asset-list.menu.show-usage': 'Vis bruk',
+  /** Menu item for deleting the asset */
+  'asset-source.asset-list.menu.delete': 'Slett',
+
+  /** Text shown in usage dialog for an image asset when there are zero, one or more documents using the *named* image **/
+  'asset-source.usage-list.documents-using-image_named_zero':
+    'Ingen dokumenter bruker bildet <code>{{filename}}</code>',
+  'asset-source.usage-list.documents-using-image_named_one':
+    'Ett dokument bruker bildet <code>{{filename}}</code>',
+  'asset-source.usage-list.documents-using-image_named_other':
+    '{{count}} dokumenter bruker bildet <code>{{filename}}</code>',
+
+  /** Text shown in usage dialog for an image asset when there are zero, one or more documents using the *unnamed* image **/
+  'asset-source.usage-list.documents-using-image_unnamed_zero':
+    'Ingen dokumenter bruker dette bildet',
+  'asset-source.usage-list.documents-using-image_unnamed_one': 'Ett dokument bruker dette bildet',
+  'asset-source.usage-list.documents-using-image_unnamed_other':
+    '{{count}} dokumenter bruker dette bildet',
+
+  /** Text shown in usage dialog for a file asset when there are zero, one or more documents using the *named* file **/
+  'asset-source.usage-list.documents-using-file_named_zero':
+    'Ingen dokumenter bruker filen <code>{{filename}}</code>',
+  'asset-source.usage-list.documents-using-file_named_one':
+    'Ett dokument bruker filen <code>{{filename}}</code>',
+  'asset-source.usage-list.documents-using-file_named_other':
+    '{{count}} dokumenter bruker filen <code>{{filename}}</code>',
+
+  /** Text shown in usage dialog for a file asset when there are zero, one or more documents using the *unnamed* file **/
+  'asset-source.usage-list.documents-using-file_unnamed_zero':
+    'Ingen dokumenter bruker denne filen',
+  'asset-source.usage-list.documents-using-file_unnamed_one': 'Ett dokument bruker denne filen',
+  'asset-source.usage-list.documents-using-file_unnamed_other':
+    '{{count}} dokumenter bruker denne filen',
+
+  /** Header in usage dialog for image assets */
+  'asset-source.asset-usage-dialog.header_image': 'Dokumenter som bruker bildet',
+  /** Header in usage dialog for file assets */
+  'asset-source.asset-usage-dialog.header_file': 'Dokumenter som bruker filen',
+  /** Text shown in usage dialog when loading documents using the selected asset */
+  'asset-source.asset-usage-dialog.loading': 'Laster',
```

Comment:
(For consistency with other UI locations)

```suggestion
  'asset-source.asset-usage-dialog.loading': 'Laster…',
```